tests/components/drag_and_drop.test.ts

```diff
@@ -148,6 +155,25 @@ describe("Drag And Drop horizontal tests", () => {
       right: 15,
     });
   });
+
+  test("Drag&Drop beyond the grid: correct col value is given to the callback", async () => {
+    freezeColumns(model, 4, sheetId);
+    setViewportOffset(model, 6 * DEFAULT_CELL_WIDTH, 6 * DEFAULT_CELL_HEIGHT);
+    await nextTick();
+    const { x: offsetCorrectionX } = model.getters.getMainViewportCoordinates();
+    const x = offsetCorrectionX + DEFAULT_CELL_WIDTH;
+    triggerMouseEvent(".o-fake-grid", "mousedown", x, 0);
+    triggerMouseEvent(".o-fake-grid", "mousemove", -50, 0);
+    const advanceTimer = edgeScrollDelay(0.1, 10);
```

Comment:
the scrolldistance you feed to edgeScrollDelay should be the same as the  scroll distance you actually apply swith the mouse move. yoseems you pur random values and iterations up until you had it working. You can just copy the same flow as the other tests

Reply:
This is slander ! I put a random value and it worked the first time :upside_down_face: